vignettes/Introduction.Rmd

```diff
@@ -326,7 +346,7 @@ The new members will be present only in instances that are created after `$set()
 
 To prevent modification of a class, you can use `lock_class=TRUE` when creating the class. You can also lock and unlock a class as follows:
 
-```{r}
+```{r, error=TRUE}
```

Comment:
Wouldn't it be better to display the error instead of commenting it out?

Currently, the output on the website:

![image](https://user-images.githubusercontent.com/11330453/161091436-a355673a-2ddd-4b53-836f-92e611df0f1d.png)

Changed output in the PR:

![image](https://user-images.githubusercontent.com/11330453/161091594-4b49dd14-c3c7-48c9-a723-e2e590dfd309.png)